tests/test_puan.py

```diff
@@ -1562,23 +1562,113 @@ def test_ndint_compress():
                 ]
             ]),
             puan.ndarray.integer_ndarray([
-                [ 0, 0, 3, 3, 1, 2],
+                [ 0, 0, 4, 3, 1, 2],
                 [ 0, 0, 0, 0, 0, 0],
                 [ 1, 2, 3, 4, 5, 6],
-                [ 0, 0, 1, 1, 1, 1],
-                [ 0, 1, 3, 2, 2, 2]
+                [ 0, 0, 4, 3, 2, 1],
+                [ 0, 2, 3, 1, 1, 1]
             ]),
             0
         ),
     ]
+
     for inpt, expected_output, axis in test_cases:
         actual_output = inpt.ndint_compress(method="rank", axis=axis)
         assert numpy.array_equal(actual_output, expected_output)
     
     with pytest.raises(ValueError):
         puan.ndarray.integer_ndarray([1, 5, 30]).ndint_compress(method="Rank")
```

Comment:
Is this check for making sure error is raised when given a method that is not in the method list? In that case maybe just a comment that that's what it is testing. Just to be clear that it has nothing to do with the [1, 5, 30] combination of numbers